test/entities.go

```diff
@@ -131,8 +131,11 @@ func (g *Blocks) New() *flow.Block {
 		g.guarantees.New(),
 	}
 
+	seals := []*flow.BlockSeal{}
```

Comment:
You can use the new `BlockSeals` generator here, similar to `g.guarantees` above.

I'd probably just add a single seal to each block:

```go
seals := []*flow.BlockSeal{
  g.seals.New(),
}
```

Thanks for adding the generator, BTW!

Reply:
@psiemens great, added that (and the corresponding instantiation) 👍